lib/config.js

```diff
@@ -1,150 +1,193 @@
 'use strict'
 
+const Immutable = require('immutable')
 const url = require('url')
 const tweetnacl = require('tweetnacl')
 
+function isRunningTests () {
+  return process.env.NODE_ENV.match(/unit|test/) &&
```

Comment:
I can remove it, but then testing configurations that are not test specific will be trickier